### PR TITLE
Fix for EXIF informations

### PR DESCRIPTION
**Pull request by mastermichi**
- Get EXIF informations in background
- Fixed a bug with showing "null"

### CODE REVIEW

File: src/com/firebirdberlin/nightdream/ui/NightDreamUI.java [hunk 1]
```diff
@@ -776,46 +772,82 @@ private double convertArcMinToDegrees(String[] separated) {
...
+                }
 
-            String tagDateTime = exif.getAttribute(ExifInterface.TAG_DATETIME);
-            if (tagDateTime != null) {
-                String[] exifDateTime = tagDateTime.split(" ");
-                String[] exifDateSplit = exifDateTime[0].split(":");
-                exifDate = exifDateSplit[2] + "." + exifDateSplit[1] + "." + exifDateSplit[0];
-                exifTime = exifDateTime[1];
-            }
+                if (!exifDate.equals("")) {
```
Comment (firebirdberlin) [inline src/com/firebirdberlin/nightdream/ui/NightDreamUI.java:801]: if (!exifDate.isEmpty())

File: src/com/firebirdberlin/nightdream/ui/NightDreamUI.java [hunk 2]
```diff
@@ -776,46 +772,82 @@ private double convertArcMinToDegrees(String[] separated) {
...
+                    exifReturnString = exifDate + "\n";
+                }
+
+                if (!exifTime.equals("")) {
```
Comment (firebirdberlin) [inline src/com/firebirdberlin/nightdream/ui/NightDreamUI.java:805]: isEmpty

File: src/com/firebirdberlin/nightdream/ui/NightDreamUI.java [hunk 3]
```diff
@@ -776,46 +772,82 @@ private double convertArcMinToDegrees(String[] separated) {
         return convert;
     }
 
-    private String getExifInformation() throws IOException, IndexOutOfBoundsException {
-        if (preloadBackgroundImageFile == null) {
-            return "";
-        }
+    private final class getExifInformation extends AsyncTask<File, Integer, String> {
```
Comment (firebirdberlin) [inline src/com/firebirdberlin/nightdream/ui/NightDreamUI.java:775]: please move this class to a separate file.

Comment (firebirdberlin) [reply src/com/firebirdberlin/nightdream/ui/NightDreamUI.java:775]: The only object from the main context which is needed is exifTextView. We could add an Interface which waits for the result and updates the textview.

File: src/com/firebirdberlin/nightdream/ui/NightDreamUI.java (hunk 2 above)
Comment (mastermichi) [reply src/com/firebirdberlin/nightdream/ui/NightDreamUI.java]: ok

Comment (mastermichi) [reply src/com/firebirdberlin/nightdream/ui/NightDreamUI.java]: ok

File: src/com/firebirdberlin/nightdream/ui/NightDreamUI.java (hunk 3 above)
Comment (mastermichi) [reply src/com/firebirdberlin/nightdream/ui/NightDreamUI.java:775]: good advice - we should open an issue for the next Version

**Pull request closed without merging**